charts/firecloudorch/templates/deployment.yaml

```diff
@@ -157,6 +157,9 @@ spec:
         livenessProbe:
           {{- toYaml .Values.probes.liveness.spec | nindent 10 }}
```

Comment:
@choover-broad I haven't seen this in any file. I just wanted to make sure that adding the start probe doesn't affect this.

Reply:
Hey @knapii-developments, adding a separate startupProbe should be fine for `firecloudorch`! It has two possible liveness probes which are enabled depending on what values are supplied.

Reply:
Actually, it looks like firecloudorch won't need a startupProbe, as it does not use liquibase. Sorry for the mixed messages @knapii-developments!

Reply:
e732039